doc/docs/1.9.x/reference/pipeline_spec.md

```diff
@@ -12,6 +12,14 @@ create pipeline](pachctl/pachctl_create_pipeline.md) section.
     "name": string
   },
   "description": string,
+  "metadata": {
+    "annotations": {
+        "foo": "bar"
```

Comment:
```suggestion
        "annotation": string
```
This is the syntax we have throughout this doc, I'm not the biggest fan of it, but might as well keep it consistent until we're ready to change it all.

Reply:
Actually do we want to make these changes at all? This is in the 1.9.x version of the docs. And these features aren't available in 1.9.x releases.

Reply:
nope, removed